master/buildbot/steps/source/repo.py

```diff
@@ -29,6 +30,14 @@
 from buildbot.steps.source.base import Source
 
 
+def executable_in_path(name):
+    for path in os.environ['PATH'].split(os.pathsep):
+        full_path = os.path.join(path, name)
+        if os.path.isfile(full_path) and os.access(full_path, os.X_OK):
+            return True
+    return False
```

Comment:
Here you are testing that pigz is available on the master, and the correct check should be on the worker